### PR TITLE
landing page test

### PR DESCRIPTION
**Pull request by giranezafiacre**
## Description

- [x] added test of landingPage.tsx

Closes https://github.com/taniarascia/takenote/issues/419
### Browser checklist

This PR has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [x] Safari

### Testing checklist

- [x] End-to-end tests have been created if necessary

### CODE REVIEW

**Comment by taniarascia** [comment]
Why are the `@resources` changed to `../../resources`? Also, a test of the landing page should test the logic, like isDemo and isMobile.

**Comment by giranezafiacre** [comment]
> Why are the `@resources` changed to `../../resources`? Also, a test of the landing page should test the logic, like isDemo and isMobile.

@taniarascia I tried to test with alias `@resources` as it was but there is an error that always appear maybe you can help,and to test logic of "ismobile" and "isDemo" I am still figuring out.
landingtest

File: config/jest.config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
   setupFilesAfterEnv: ['@testing-library/jest-dom', 'jest-extended'],
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
   moduleNameMapper: {
+    '^[@./a-zA-Z0-9$_-]+\\.(png|svg|gif)$': '<rootDir>/tests/__mocks__/styleMock.ts',
```
Comment (kodyclemens) [inline config/jest.config.js:13]: Why was this added? The map is never used and seems unnecessary.

File: tests/unit/client/Landing.test.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import * as rdd from 'react-device-detect'
+
+import { LandingPage } from '@/components/LandingPage'
+
+describe('<LandingPage />', () => {
+  it('renders the LandingPage component', () => {
+    render(<LandingPage />)
+  })
+  it('<LandingPage />in mobile view', () => {
+    Object.assign(rdd, { isMobile: true })
```
Comment (kodyclemens) [inline tests/unit/client/Landing.test.tsx:12]: This assign is doing nothing except making a new object that is not used. With this approach I assume you are trying to [mock an import](https://dev.to/lmeromy/testing-react-components-how-to-mock-imports-with-jest-1k90) in `<LandingPage />`?

**Review by kodyclemens — CHANGES_REQUESTED** [review]

**Comment by taniarascia** [comment]
Closing due to inactivity.

**Pull request closed without merging**